### PR TITLE
fix action attempts test

### PR DESCRIPTION
**Pull request by kainpets**

### CODE REVIEW

File: src/main/java/com/seam/api/resources/connectedaccounts/types/ConnectedAccountsGetRequest.java [hunk 1]
```diff
@@ -66,6 +67,14 @@ public static ConnectedAccountsGetRequest of(ConnectedAccountsGetRequestEmail va
         return new ConnectedAccountsGetRequest(value, 1);
     }
 
+    public static ConnectedAccountsGetRequest id(String id) {
```
Comment (dsinghvi) [inline src/main/java/com/seam/api/resources/connectedaccounts/types/ConnectedAccountsGetRequest.java:70]: getById, can you make this non static

File: src/main/java/com/seam/api/SeamBuilder.java [hunk 2]
```diff
@@ -19,6 +20,11 @@ public final class SeamBuilder {
 
     private Environment environment = Environment.DEFAULT;
 
+    public SeamBuilder apiKey(String apiKey) {
+        this.clientOptionsBuilder.addHeader("Authorization", "Bearer " + apiKey);
+        return this;
+    }
```
Comment (dsinghvi) [inline src/main/java/com/seam/api/SeamBuilder.java:26]: @kainpets why did you add this?

Comment (kainpets) [reply src/main/java/com/seam/api/SeamBuilder.java:26]: The testUtils fails without it at line 45

Comment (kainpets) [reply src/main/java/com/seam/api/SeamBuilder.java:26]: It's copied from the previous version

**Comment by kainpets** [comment]
Included in newer regeneration
